runtime/sema/errors.go

```diff
@@ -4231,6 +4242,10 @@ func (e *UnrepresentableEntitlementMapOutputError) Error() string {
 	return fmt.Sprintf("cannot map %s through %s because the output is unrepresentable", e.Input.AccessKeyword(), e.Map.QualifiedString())
 }
 
+func (e *UnrepresentableEntitlementMapOutputError) SecondaryError() string {
+	return fmt.Sprintf("this usually occurs because the input set is disjunctive and %s is one-to-many", e.Map.QualifiedString())
```

Comment:
```suggestion
	return fmt.Sprintf(
		"this usually occurs because the input set is disjunctive and %s is one-to-many", 
		e.Map.QualifiedString()
	)
```

Maybe also chop the arguments of the other `fmt.Sprintf` calls to reduce the line length and make it easier to read what is filled into the format string at what positions